sdk/examples/wallet/ledger_nano.rs

```diff
@@ -38,11 +38,11 @@ async fn main() -> Result<()> {
     // This example uses secrets in environment variables for simplicity which should not be done in production.
     dotenvy::dotenv().ok();
 
-    let client_options = ClientOptions::new().with_node(&var("NODE_URL").unwrap())?;
+    let client_options = ClientOptions::new().with_node(&std::env::var("NODE_URL").unwrap())?;
```

Comment:
Added but not removed

Reply:
thanks clippy :/

Reply:
Done (if not in this ... then in some other universe)